Source/WebCore/css/svg.css

```diff
@@ -61,8 +61,12 @@ text, foreignObject {
     display: block
 }
 
-text, tspan, textPath {
-   white-space: nowrap
+text {
+   white-space: nowrap;
+}
+
+tspan, textPath {
+    white-space: inherit;
 }
```

Comment:
nit: inconsistent indentation

Reply:
@nt1m - fixed in https://github.com/WebKit/WebKit/commit/6a01dcc2a3e136b2acf2af78ae46bf8d80045f5e.